cmd/push.go

```diff
@@ -75,11 +75,11 @@ var pushCmd = &cobra.Command{
 				checkError(err, "")
 			}
 
-			var asFile bool
-			if sourceType == "binary" {
-				asFile = true
+			var path string
+			if sourceType == "local" {
```

Comment:
this looks odd, what will happen with path when `sourceType` is binary?

Reply:
shouldn't we call PushLocal where the last argument is slice with one file and that is the binary?

Reply:
that was my stupid mistake :( and was breaking the binary thing